### ISSUE TITLE
FAH v8.4.2 on Windows, System Tray icon disappeared allowing second instance

### ISSUE THREAD

**Issue by Hou5e** [FoldingAtHome/fah-client-bastet#269]
With FAH v8.4.2 on Win10, the Windows System Tray notify icon for FAH was gone (it ran for a week and died at some point in the last few days with no pop-up error messages), so I started FAH up again, only to find I had 2 copies running with Task Manager (there is only 1 GPU on this PC):
![image](https://github.com/user-attachments/assets/11eaf4f7-eaa0-4232-a3d6-d43c1075b57e)

Since the system tray program wasn't running, FAH didn't prevent multiple instances from being run. When FAH started up the second copy it should probably check if it is already running and try to prevent that (even if it's a message that says FAH is already running, do you want to continue to run another copy).
```
19:28:48:I1:WU14223:Completed 0 out of 1250000 steps (0%)
19:28:50:I1:WU14223:Checkpoint completed at step 0
19:30:11:I1:WU14223:Completed 12500 out of 1250000 steps (1%)
19:31:32:I1:WU14223:Completed 25000 out of 1250000 steps (2%)
19:31:34:I1:WU14223:Checkpoint completed at step 25000
19:32:56:I1:WU14223:Completed 37500 out of 1250000 steps (3%)
19:34:16:I1:WU14223:Completed 50000 out of 1250000 steps (4%)
19:34:18:I1:WU14223:Checkpoint completed at step 50000
19:35:38:I1:WU14223:Completed 62500 out of 1250000 steps (5%)
19:36:59:I1:WU14223:Completed 75000 out of 1250000 steps (6%)
19:37:01:I1:WU14223:Checkpoint completed at step 75000
*********************** Log Started 2024-08-12T19:37:02Z ***********************
19:37:02:I1:*********************** Folding@home Client ***********************
19:37:02:I1:    Version: 8.4.2
19:37:02:I1:     Author: Joseph Coffland <joseph@cauldrondevelopment.com>
...
19:38:04:I1:WU14224:Checkpoint completed at step 0
...
19:38:44:I1:WU14223:Completed 87500 out of 1250000 steps (7%)
...
19:40:38:I1:WU14224:Completed 12500 out of 1250000 steps (1%)
```
Log file: [log-20240812-194157.txt](https://github.com/user-attachments/files/16599494/log-20240812-194157.txt)

**Comment by jcoffland** [comment]
v7 solved this by taking an exclusive lock on the client DB.  I'll add this to v8.

**Comment by jcoffland** [comment]
Fixed in v8.4.4.

**Comment by jcoffland** [comment]
Are you sure the systray icon was really gone when you ran the second F@H instance?  On Windows, sometimes the systray icons will be hidden by the OS and you have to click the up chevron icon to see the hidden icons.

![screenshot-20240830-174052](https://github.com/user-attachments/assets/8c378553-d92c-49a7-a930-7f8dfb0ba02f)

Perhaps you know this already.  I'm just trying to eliminate explanations for the disappearing systray icon.
